packages/backend/README.md

```diff
@@ -39,6 +39,21 @@ You can also, instead of using dummy values for a huge number of environment var
 
 The backend starts up on port 7000 per default.
 
+### Debugging
+
+The backend is a node process that can be inspected to allow breakpoints and live debugging. To enable this, pass the `--inspect` flag to [backend:dev](https://backstage.io/docs/cli/commands#backenddev).
+
+To debug the backend in [Visual Studio Code](https://code.visualstudio.com/):
+
+- Enable Auto Attach (⌘ + Shift + P > Toggle Auto Attach > Only With Flag)
+- Open a VSCode terminal (Control + `)
+- Run the backend from the VSCode terminal:
+
+```
+$ cd packages/backend
+$ yarn backstage-cli backend:dev --inspect
```

Comment:
would probably be more familiar, I think, to just make this `yarn start-backend --inspect` from the root - or does not the flag get passed along then?

Reply:
I tried this initially and had the debugger trying to latch on to intermediate steps (e.g. `yarn start-backend` gets a debugger, `backstage-cli backend:dev` gets another). That must have been with a different auto attach mode though, seems to work as expected when I tried it now. Updated!

Reply:
Yeah that mode was smart - i didn't know if it, neat